driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/unified/ChangeStreamsTest.java

```diff
@@ -19,18 +19,59 @@
 import com.mongodb.lang.Nullable;
 import org.bson.BsonArray;
 import org.bson.BsonDocument;
+import org.junit.After;
 import org.junit.runners.Parameterized;
 
 import java.io.IOException;
 import java.net.URISyntaxException;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
+
+import static com.mongodb.reactivestreams.client.syncadapter.SyncMongoClient.disableSleep;
+import static com.mongodb.reactivestreams.client.syncadapter.SyncMongoClient.enableSleepAfterCursorOpen;
+import static org.junit.Assume.assumeFalse;
 
 public final class ChangeStreamsTest extends UnifiedReactiveStreamsTest {
+
+    @SuppressWarnings("ArraysAsListWithZeroOrOneArgument")
+    private static final List<String> ERROR_REQUIRED_FROM_CHANGE_STREAM_INITIALIZATION_TESTS =
+            Arrays.asList(
```

Comment:
We can just use `Collections.singletonList`. If in the future we need to specify multiple tests here, we will be forced to replace it with `Arrays.asList`. On the other hand, if we specify multiple array elements in `Arrays.asList` in the future, nothing will force us to remove the `@SuppressWarnings("ArraysAsListWithZeroOrOneArgument")` annotation, which will become useless. That's why I propose to use `Collections.singletonList`.

Reply:
Fair enough.  Done.